app/redux/errorHandlerMiddleware.ts

```diff
@@ -3,7 +3,7 @@ import { AnyAction, Middleware } from 'redux';
 const createErrorHandlerMiddleware = (onError: (error: Error) => AnyAction): Middleware => () => (next) => (action) => {
   try {
     const result = next(action);
-    if (result.then && result.catch) {
+    if (result?.then && result?.catch) {
```

Comment:
Maybe we can somehow to handle this behaviour? If not right now, just add @todo for this line as backlog issue.

Reply:
Yep, for example, accept the rule that all thunk actions should return something :)